scripts/generate_export.py

```diff
@@ -489,21 +491,18 @@ def sequences_overlap(true_seq, pred_seq):
 
     for es in exception_submissions:
         exception_ids.append(int(es.id))
-
-    exception_ids_df = pd.DataFrame(exception_ids, columns=["Submission ID"])
-
-    for exception_submission in exception_submissions:
-        exception_filenames.append(str(exception_submission.input_filename))
-
-    exception_filenames_df = pd.DataFrame(exception_filenames, columns=["File Name"])
-    exceptions_df = pd.concat([exception_filenames_df, exception_ids_df], axis=1)
-    for exception_submission in exception_submissions:
-        result = indico_wrapper.get_submission_results(exception_submission)
+        print(exception_ids)
```

Comment:
Are you sure you want to keep print statements in here?

Reply:
Nope. They can be removed. Can you disapprove certain changes like these for which you commented? Or do I need to push changes again and create a PR?

Reply:
Next steps is making changes on your branch locally. Then commit and push the changes and this PR will get updated with your new code that I can review.

Reply:
Sorry, I should have had the steps more formalized.  If you make the changes to this current branch the PR will update so you don't need to make a new (or a new branch).

from the solutions-toolkit repo on your local
1) `git checkout Gnana_changes_3_5_21` (this branch)
2) make requested edits above
3) git add + commit
4) git push 

This current PR will reflect the new changes.